iozone/client.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+# enable some more output
+set -x
+
+[ -n "${SERVER}" ]
+[ -n "${EXPORT}" ]
+
+# install build and runtime dependencies
+yum -y install git gcc nfs-utils time
+
+mkdir -p /mnt/ganesha
+
+cd ~
+curl -o iozone3_394.tar http://www.iozone.org/src/current/iozone3_394.tar
+
+tar xvf iozone3_394.tar 
+cd iozone3_394/src/current
+make
+make linux
+ioZoneDir=$(pwd)/iozone
+
+mount -t nfs -o vers=3 ${SERVER}:${EXPORT} /mnt/ganesha
+
+cd /mnt/ganesha
+
+echo "Running Iozone Test On NFSv3 "
+echo "+++++++++++++++++++++++++++++"
+
+$ioZoneDir -a > ../ioZoneLog.txt
+
+grep "iozone test complete" ../ioZoneLog.txt;
+
+ret=$?
+
+if [ $ret -eq 0 ]
+then
+        echo "IOZone Test Is Completed And Successful";
```

Comment:
Can the success lines also say which NFS version is in use?

Reply:
@dang Ok...I will fix it .. :)